coretex/cli/commands/node.py

```diff
@@ -41,13 +43,11 @@ def start(image: Optional[str]) -> None:
         ):
             return
 
-        node_module.stop()
+        node_module.stop(config["nodeId"])
```

Comment:
This will raise an error if `nodeId` does not exist in configuration. Use Node name to fetch the Node id if it isn't saved in the config.